contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -607,7 +642,7 @@ CREATE OR REPLACE VIEW information_schema_tsql.COLUMN_DOMAIN_USAGE AS
            CAST(isc_col."TABLE_NAME" AS sys.sysname),
            CAST(isc_col."COLUMN_NAME" AS sys.sysname)
 
-    FROM information_schema_tsql.columns AS isc_col
+    FROM information_schema_tsql.columns_internal AS isc_col
```

Comment:
why this change?

Reply:
Seems better to fetch directly from the internal view instead of the wrapper. Just for consistency.